JavaScript/Github_Fetch_Issue_Details/issueDetails.js

```diff
@@ -0,0 +1,67 @@
+const express = require("express");
+const axios = require("axios");
+const fs = require("fs");
+const getDetails = async (username, repoName) => {
+  const url = `https://api.github.com/repos/${username}/${repoName}/issues`;
+  await axios
+    .get(url)
+    .then(async (result) => {
+      console.log(result);
```

Comment:
Any specific reason why the result is logged to console? If it is for the user, please put an apt message along with it :)

Reply:
Two reasons:
- For Testing Purpose
- Will help someone understand the code, In case anyone wants to extend it or soemthing.

Reply:
Hi @Naman-1234 

Appreciate your understanding on this. The testing purpose is absolutely not needed but for understanding you can put up a comment here.

Reply:
Sure, Will replace it with a comment in the latest commit.